packages/auto-label/test/language-path-tests.ts

```diff
@@ -94,13 +94,6 @@ describe('language-and-path-labeling', () => {
         .get('/repos/testOwner/testRepo/pulls/12/files')
         .reply(200, pr_files_payload)
 
-        // Mock issues.addlabels adding path_label
-        .post('/repos/testOwner/testRepo/issues/12/labels', body => {
```

Comment:
Do you know why this assertion ended up needing to be dropped, this concerns me slightly?

Reply:
Yes, sorry about this - there are 2 redundant calls to `context.octokit.issues.addLabels` within same function, I just united those into single call... It might be confusing, I just get used to approach to fix things once you encounter them.